src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx

```diff
@@ -0,0 +1,154 @@
+
+## Solution
+
+Verify that the New Relic infrastructure agent endpoints are reachable from the host where it is installed.
+Follow the instructions below to confirm there are no issues with your environment's network settings:
+
+<CollapserGroup>
+  
+<Collapser id="windows" title="Windows">
+
+Run the following command on the terminal:
+```
+$ Invoke-WebRequest -Uri "https://infra-api.newrelic.com/cdn-cgi/trace"
+```
+
+The expected output should include a `200 (OK)` as shown below:
+```
+StatusCode        : 200
+StatusDescription : OK
+Content           : fl=367f145
+                    h=infra-api.newrelic.com
+                    ip=XX.YY.ZZ.ZZ
+                    ts=1696347189.607
+                    visit_scheme=https
+                    uag=Mozilla/5.0 (Windows NT; Windows NT 10.0; en-US) WindowsPowerShell/5.1.22621.963
+                    colo=MAD
+                    sliver=none
+                    htt...
+RawContent        : HTTP/1.1 200 OK
+                    Transfer-Encoding: chunked
+                    Connection: keep-alive
+                    Access-Control-Allow-Origin: *
+                    CF-RAY: 81063def0905384e-MAD
+                    X-Frame-Options: DENY
+                    X-Content-Type-Options: nosniff
+                    Cache-Control...
+Forms             : {}
+Headers           : {[Transfer-Encoding, chunked], [Connection, keep-alive], [Access-Control-Allow-Origin, *], [CF-RAY, 81063def0905384e-MAD]...}
+Images            : {}
+InputFields       : {}
+Links             : {}
+ParsedHtml        : System.__ComObject
+RawContentLength  : 284
+```
+
+</Collapser>
+
+<Collapser id="linux" title="Linux">
+
+1. **Resolve the domain:** Use nslookup or dig to validate that the Linux system can resolve the domain to an IP address:
+
+```
+$ nslookup infra-api.newrelic.com
+```
+
+The output should contain a name and IP address.
+
+2. **Ping the server:** Try pinging the IP address obtained above to ensure you can reach the server.
+
+```
+$ ping 162.247.241.2
+```
+
+The server should be reachable.
+
+3. **Traceroute:** Use traceroute to check the network path between your system and the server:
+
+```
+$ traceroute -I infra-api.newrelic.com
+```
+
+Analyze the output to identify potential network issues, such as high latency or packet loss.
+
+4. **Curl:** Use curl to check if the endpoint is reachable:
+
+```bash
+$ curl -v https://infra-api.newrelic.com/cdn-cgi/trace
+
+fl=366f101
+h=infra-api.newrelic.com
+ip=XX.YY.ZZ.ZZ
+ts=1696347098.76
+visit_scheme=https
+uag=curl/7.88.1
+colo=MAD
+sliver=none
+http=http/1.1
+loc=ES
+tls=TLSv1.3
+sni=plaintext
+warp=off
+gateway=off
+rbi=off
+kex=X25519
+* Connection #0 to host infra-api.newrelic.com left intact
+```
+
+</Collapser>
+
+</CollapserGroup>
+
+## Troubleshoot with NRDiag
+
+[Code Implementation](https://github.com/newrelic/newrelic-diagnostics-cli/blob/main/tasks/infra/agent/connect.go#L48)
```

Comment:
@josemore What's the purpose of this link? Can you add something before this link, like what readers would expect to see from this link or why they'd want to go to it?